src/internal/connector/exchange/delta_get.go

```diff
@@ -1,121 +0,0 @@
-package exchange
```

Comment:
This is deleted because it was hacking around functionality that was missing in v40, right?